### PR TITLE
Manually deploy bases

### PR DESCRIPTION
**Pull request by 0xGabi**
Fix https://github.com/aragon/aragon-cli/issues/1331

Add bases (ENS, APMRegistry and DAOFactory) to back-end and refactor logic to use on other places.

### CODE REVIEW

File: buidler.config.js [hunk 1]
```diff
@@ -1,4 +1,14 @@
 module.exports = {
+  network: {
```
Comment (ajsantander) [inline buidler.config.js]: This is the root buidler.config.js. At this level, buidler is just used to compile contracts and not actually used to interact with a chain, so I don't think that we should tinker with network options at this level. It should work without this change.

**Comment by ajsantander** [comment]
Pls run `npm run lint` and make sure all linting passes. We're using eslint + prettier for typescript. Is your editor picking these up?

**Comment by ajsantander** [comment]
I see that projects/token-wrapper is still using the localhost network. Have you tested the plugin in this project as well? Remember that this one requires you to build the plugin first.

File: src/tasks/start/utils/backend/bases/apm.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import { hash as namehash } from 'eth-ens-namehash'
```
Comment (ajsantander) [inline src/tasks/start/utils/backend/bases/apm.ts]: We can get rid of the bases directory. These files should live at the same level as app.ts, dao.ts, etc.

**Comment by ajsantander** [comment]
Tests are not passing. Please review that existing tests pass and add new tests for new code. 

You can leave new tests to me, since I'm working on the task of increasing test coverage anyway. If you pass them to me tho, pls add placeholders for them :pray: . One test file per source file with a dummy "  it.skip('more tests needed')" should be enough.

**Comment by ajsantander** [comment]
npm run build is failing. I think this has to do with the tests as well.

**Comment by ajsantander** [comment]
After making sure tests pass, also please run `npx buidler start` in both the counter and the token-wrapper test projects. What I usually do is:
1) Test front end hot reloading by making a change to App.js.
2) Test back end hot reloading by making a change to the app's main contract.
3) *Optional* try changing script.js for a full client reload.

**Comment by ajsantander** [comment]
Regarding
> "Fix issue with APM DAO not detected by the wrapper. As a consequence the ENS dao address is not resolved. Related to aragon/aragon-cli#1082"

If this is also happening with DAOs created by the aragon-cli, then this is not a concern of this PR.

**Comment by ajsantander** [comment]
Pls update travis.yml to not install and run aragen also, since this will no longer be needed after this PR.

File: src/tasks/start/utils/backend/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { BuidlerPluginError } from '@nomiclabs/buidler/plugins'
```
[the remark this answers is not in the document]
Comment (dapplion) [reply src/tasks/start/utils/backend/helpers.ts]: What about `logs-helpers.ts`? When I read the code first I was confused what `logs` file was doing

**Comment by dapplion** [comment]
Submitted this PR which solves the issues outline above https://github.com/aragon/buidler-aragon/pull/8

**Comment by 0xGabi** [comment]
@dapplion Thanks for the review and further address yourself the concerns 🙏

**Pull request merged**